packages/orbit-components/src/hooks/useRandomId/index.ts

```diff
@@ -1,14 +1,17 @@
-import { useUIDSeed, useUID } from "react-uid";
+import { useRandomId as useRandomIdContext } from "../../OrbitProvider/RandomId/Provider";
 
 type UseRandomIdSeed = () => (id: string) => string;
 type UseRandomId = () => string;
 
-export const useRandomIdSeed: UseRandomIdSeed = () => {
-  return useUIDSeed();
+export const useRandomId: UseRandomId = () => {
+  const useId = useRandomIdContext();
+  return useId();
```

Comment:
Typescript doesn't like this.
> Cannot invoke an object which is possibly 'undefined'

Do we need to assert that it is defined before calling it? 💭 
In practice it should never happen because of the thrown error, but I guess TS doesn't catch that 🤔